src/components/DateRangePicker.jsx

```diff
@@ -51,6 +51,8 @@ const defaultProps = {
   // input related props
   startDatePlaceholderText: 'Start Date',
   endDatePlaceholderText: 'End Date',
+  startDateOffset: undefined,
+  endDateOffset: undefined,
```

Comment:
Please add a test for DateRangePicker

Reply:
@ljharb, thanks, I've added tests for the passing of `startDateOffset` and `endDayOffset` to ensure they're received by `DayPickerRangeController`.
The tests within `DayPickerRangeController` should cover the date offset functionality